### PR TITLE
Bug fix box plots

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds: 
- showing sample ids when hoovering over a qc-plot
- showing functions when hoovering over XY plot

**Review:**
- [ ] code approved by
- [x] tests executed by @mayabrandi 
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Bump version
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by mayabrandi** [comment]
Skärmavbild 2021-07-13 kl  15 30 01
Skärmavbild 2021-07-13 kl  15 29 32
Skärmavbild 2021-07-13 kl  15 29 27

File: statina/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-
+__version__ = "1.2.5"
```
Comment (Mropat) [inline statina/__init__.py:1]: If you add it here, you also need to add the path to it in .bumpversion.conf

Comment (Mropat) [reply statina/__init__.py:1]: So that the CI actions can fund all places where we keep version and bump it

File: .bumpversion.cfg [hunk 2]
```diff
@@ -5,3 +5,4 @@ tag = True
 tag_name = v{new_version}
 
 [bumpversion:file:setup.py]
+[bumpversion:file:statina.__init__.py]
```
Comment (Mropat) [inline .bumpversion.cfg:8]:
```suggestion
[bumpversion:file:statina/__init__.py]
```

**Pull request merged**